ci/run-check.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# Enable warnings about unused extern crates
+# export RUSTFLAGS=" -W unused-extern-crates"
+rustc --version
+rustup --version
+cargo --version
+
+case $TARGET in
+  cargo-build)
+    cargo build --release "$@"
+    ;;
+
+  test-general)
+    RUST_MIN_STACK=8388608 cargo test --workspace --release --features runtime-benchmarks,try-runtime --exclude runtime-integration-tests
+    ;;
+
+  test-integration)
+    RUST_MIN_STACK=8388608 cargo test --release --package runtime-integration-tests --features fast-runtime
+    ;;
```

Comment:
It would be cool to test it in `debug` mode. The `RUST_MIN_STACK` is not needed now.

Reply:
We can not test it in debug mode because it takes to much time. We need to fix the integration-test timing before